sciLifeLab_utils/run_denovo.py

```diff
@@ -184,7 +184,7 @@ def report(state, **kwargs):
 
 def main():
     try:
-        conf_file = os.path.join(os.environ.get('HOME'), '.nougat','scilifelab.conf')
+        conf_file = os.environ.get("NOUGAT_CONF", os.path.join(os.environ.get('HOME'), '.nougat','scilifelab.conf'))
```

Comment:
Sweet, thanks :D

On Thu, Jun 30, 2016 at 2:15 PM, Remi-Andre Olsen <notifications@github.com>
wrote:

> In sciLifeLab_utils/run_denovo.py
> <https://github.com/SciLifeLab/NouGAT/pull/43#discussion_r69121033>:
>
> > @@ -184,7 +184,7 @@ def report(state, **kwargs):
> >
> >  def main():
> >      try:
> > -        conf_file = os.path.join(os.environ.get('HOME'), '.nougat','scilifelab.conf')
> > +        conf_file = os.environ.get("NOUGAT_CONF", os.path.join(os.environ.get('HOME'), '.nougat','scilifelab.conf'))
>
> @sylvinite <https://github.com/sylvinite> Just use this variable and
> you'll be good to go.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SciLifeLab/NouGAT/pull/43/files/53c332606af4793fa84966259d7992a46de74dd4#r69121033>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFMx8GAikj9qvH8SsHsR0L9p5LMrJ4knks5qQ7NVgaJpZM4JCGEe>
> .
>



-- 
Isak Sylvin
Bioinformatician
Science for Life Laboratory
[image: SciLifeLab-logo-greenliten]
isak.sylvin@scilifelab.se <irene.anderson@scilifelab.se>
Phone: +46 (0)708-406453